editor/import/scene_import_settings.cpp

```diff
@@ -1144,18 +1280,43 @@ void SceneImportSettingsDialog::_notification(int p_what) {
 		} break;
 
 		case NOTIFICATION_THEME_CHANGED: {
-			action_menu->begin_bulk_theme_override();
-			action_menu->add_theme_style_override("normal", get_theme_stylebox("normal", "Button"));
-			action_menu->add_theme_style_override("hover", get_theme_stylebox("hover", "Button"));
-			action_menu->add_theme_style_override("pressed", get_theme_stylebox("pressed", "Button"));
-			action_menu->end_bulk_theme_override();
+			batch_menu->begin_bulk_theme_override();
+			batch_menu->add_theme_style_override("normal", get_theme_stylebox("normal", "Button"));
+			batch_menu->add_theme_style_override("hover", get_theme_stylebox("hover", "Button"));
+			batch_menu->add_theme_style_override("pressed", get_theme_stylebox("pressed", "Button"));
+			batch_menu->end_bulk_theme_override();
 
 			if (animation_player != nullptr && animation_player->is_playing()) {
 				animation_play_button->set_icon(get_editor_theme_icon(SNAME("Pause")));
 			} else {
 				animation_play_button->set_icon(get_editor_theme_icon(SNAME("MainPlay")));
 			}
 			animation_stop_button->set_icon(get_editor_theme_icon(SNAME("Stop")));
+
+			for (int i = 0; i < TOOL_MAX; i++) {
+				static const char *icons[TOOL_MAX] = {
+					"ToolSelect",
+					"ListSelect",
+					"NodeSelect",
+					"MeshSelect",
+					"MaterialSelect",
+					"StandardMaterial3D",
+				};
+				tool_buttons[i]->set_icon(get_theme_icon(icons[i], "EditorIcons"));
```

Comment:
```suggestion
				tool_buttons[i]->set_icon(get_editor_theme_icon(icons[i]));
```

We have a special method now, to safe on string names.

Reply:
Oh, nice, will use this.